unit-tests/test-coldlaunch.js

```diff
@@ -112,15 +112,14 @@ describe('cold-launch', () => {
 
     // --app not provided throws an error
     it('error thrown when --app not provided', () => {
-
       let errorFunc = () => {
+
         let options = {
           'metrics': 'metrics.ldjson',
           'runs': 30
         };
         let coldlaunch = new ColdLaunch(options);
       }
-
```

Comment:
Yes, sorry. I meant a line after the entire declaration, not inside the function. There should be no gap between the start of a function and its declarations.